chat-widget/src/components/livechatwidget/common/getMockChatSDKIfApplicable.ts

```diff
@@ -0,0 +1,21 @@
+import { DemoChatSDK } from "../../webchatcontainerstateful/common/DemoChatSDK";
+import { DesignerChatSDK } from "../../webchatcontainerstateful/common/DesignerChatSDK";
+import { MockChatSDK } from "../../webchatcontainerstateful/common/mockchatsdk";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const getMockChatSDKIfApplicable = (chatSDK: any, type?: string) => {    
+    if (type) {
+        switch(type.toLowerCase()) {
+            case "demo":
+                chatSDK = new DemoChatSDK();

```

Comment:
add return, no need to continue evaluation once the work is done

Reply:
It's in a `switch` statement, every case has a `break`. So, only the applicable scenario would be executed then return the proper `chatSDK` accordingly